### PR TITLE
Move inline script to template.js [isis]

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Keep javascript in .js files

At the end of index.php in isis there is an inline script. We can do better than that: Put those lines in template.js!
##### Testing

Apply the patch go to admin area (isis should be the template)
Check that everything still in order!
Also check the options:
Collapse Administrator Menu
Display Header
Pinned Toolbar
and Status Module Position
#### B/C

NONE

### CODE REVIEW

**Comment by n9iels** [comment]
@test works fine for me, thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6696">issues.joomla.org/joomla-cms/6696</a>.</sub>

**Comment by RickR2H** [comment]
@test Test works for me.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6696">issues.joomla.org/joomla-cms/6696</a>.</sub>

**Comment by zero-24** [comment]
RTC Thanks for testing :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6696">issues.joomla.org/joomla-cms/6696</a>.</sub>

**Comment by zero-24** [comment]
and coding @dgt41 :+1:

**Comment by dgt41** [comment]
@phproberto Can I ask to re evaluate the decision for 3.5? 
The reason for that request is that this PR solves #5970

File: administrator/templates/isis/js/template.js [hunk 1]
```diff
@@ -56,6 +56,44 @@
 		});
 
 		/**
+		 * USED IN: All views with toolbar and sticky bar enabled
+		 */
+		var navTop;
+		var isFixed = false;
+
+		if (typeof stickyToolbar != 'undefined' && stickyToolbar == 1) {
```
Comment (Fedik) [inline administrator/templates/isis/js/template.js]: as `stickyToolbar` is a global variable, you could do a lot simpler: `if(window.stickyToolbar){ ... }` :wink: 

other possible issue, that other extension use variable with same name somewhere, so  then maybe better  would be `joomlaIsisStickyToolbar` and `joomlaIsisOffsetTop`, also will be more simple to recognize what is it :wink:

idealy would be `$doc->addScriptOptions()` ... but we do not have it :alien:

Comment (Fedik) [inline administrator/templates/isis/js/template.js]: ah it already RTC, sorry! missed that ... can ignore my comment

Comment (dgt41) [inline administrator/templates/isis/js/template.js]: @Fedik Thanks for the input. It doesn’t really matter that it was RTC as the milestone is 3.5, so we can improve it before it gets merged! I made the changes suggested, of without the `$doc->addScriptOptions()`. That has to wait 😉

**Comment by wilsonge** [comment]
Merged into 3.5 development - sorry but I agree with Roberto in this case.

**Pull request closed without merging**